app-crypt/swtpm/swtpm-0.3.1-r1.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
```

Comment:
Eh? No python3_8? ;-)

Reply:
Not sure what happened there. Fixed now.